lib/assets/javascripts/cable.js.coffee

```diff
@@ -5,4 +5,8 @@
   PING_IDENTIFIER: "_ping"
 
   createConsumer: (url) ->
+    token = $("meta[name=csrf-token]").attr("content")
```

Comment:
The current documentation for creating a consumer has it created immediately (ie, not when DOM is ready or similar) and by default in a newly generated rails app the tokens appear after the javascript include tag, so by default the $("meta[name=csrf-token]") will not exist when this code executes.

I think that we will need to either:
* update docs to create the consumer after the DOM is ready, or
* update docs to include moving `csrf_meta_tags` to before `javascript_include_tag`

I think the second option is better because it would mean that the consumer would be set up earlier.

Also - This adds a dependency on jquery (or other javascript library with `$`). Not sure if we should try to avoid the dependency but we should at least document it if we have it.